docs/ref/middleware.txt

```diff
@@ -158,12 +158,22 @@ It will NOT compress content if any of the following are true:
 You can apply GZip compression to individual views using the
 :func:`~django.views.decorators.gzip.gzip_page()` decorator.
 
+.. note::
+
+    If the response to be compressed has an ``ETag`` header, the header will
+    be changed to a weak ``ETag`` to comply with :rfc:`7232`.
+
 .. versionchanged:: 1.10
 
     In older versions, Django's CSRF protection mechanism was vulnerable to
     BREACH attacks when compression was used. This is no longer the case, but
     you should still take care not to compromise your own secrets this way.
 
+.. versionchanged:: 1.11
+
+    In older versions, the ``ETag`` header was changed in a way that did not
```

Comment:
I'll leave it up to you. It seems like a minor detail that probably won't cause confusion without an annotation,but I haven't used any of the conditional response stuff.